ledger-tool/src/main.rs

```diff
@@ -1080,7 +1087,7 @@ fn main() {
                     Arg::with_name("record_slots_config")
                         .long("record-slots-config")
                         .default_value("hash-only")
-                        .possible_values(&["hash-only", "accounts"])
+                        .possible_values(&["hash-only", "accounts", "tx", "tx-accounts"])
```

Comment:
I think making this take multiple values might be a more extensible format. For example, use:
```
--record-slots-config accounts --record-slots-config transactions
```
I can think of one other item that we may want to add right now (block cost), and allowing the specification of multiple items avoids a growing matrix of options

Reply:
Nice idea. :+1: I've changed it.